src/assets/main.css

```diff
@@ -32,4 +32,4 @@ a,
     grid-template-columns: 1fr 1fr;
     padding: 0 2rem;
   }
-}
+} */
```

Comment:
Why was this file commented out, if you want to comment out multiple lines, please explain why.